src/bots/depositor.py

```diff
@@ -126,55 +127,29 @@ def execute(self, block: BlockData) -> bool:
 
         return True
 
-    def _is_mellow_depositable(self, module_id: int) -> bool:
-        if not variables.MELLOW_CONTRACT_ADDRESS:
-            return False
-        try:
-            buffered = self.w3.lido.lido.get_buffered_ether()
-            unfinalized = self.w3.lido.lido_locator.withdrawal_queue_contract.unfinalized_st_eth()
-            if buffered < unfinalized:
-                return False
-            staking_module_contract: StakingModuleContract = self.w3.lido.simple_dvt_staking_strategy.staking_module_contract
-            if staking_module_contract.get_staking_module_id() != module_id:
-                logger.debug(
-                    {
-                        'msg': 'Mellow module check failed.',
-                        'contract_module': staking_module_contract.get_staking_module_id(),
-                        'tx_module': module_id,
-                    }
-                )
-                return False
-            balance = self.w3.lido.simple_dvt_staking_strategy.vault_balance()
-        except Exception as e:
-            logger.warning(
-                {
-                    'msg': 'Failed to check if mellow depositable.',
-                    'module_id': module_id,
-                    'err': repr(e),
-                }
-            )
-            return False
-        MELLOW_VAULT_BALANCE.labels(module_id).set(balance)
-        if balance < variables.VAULT_DIRECT_DEPOSIT_THRESHOLD:
-            logger.info(
-                {
-                    'msg': f'{balance} is less than VAULT_DIRECT_DEPOSIT_THRESHOLD while building mellow transaction.',
-                    'balance': balance,
-                    'threshold': variables.VAULT_DIRECT_DEPOSIT_THRESHOLD,
-                }
-            )
-            return False
-        logger.debug({'msg': 'Mellow module check succeeded.', 'tx_module': module_id})
-        return True
-
     def _check_balance(self):
```

Comment:
`check_balance` is only a depositor function, so we will lose updates to metrics in other bots, is it intended?